sod-server/src/commands/base/PlaceRoadCommand.ts

```diff
@@ -14,3 +14,18 @@ export class PlaceRoadCommand extends Command<MyRoom, Payload> {
     }
   }
 }
+
+export class PlaceRoadWithPayCommand extends Command<MyRoom, Payload> {
+  execute(payload: Payload) {
+    const player = this.room.state.players.get(payload.playerId)
+    const availableRoad = player.roads.find((x) => !x.edge)
+    if (availableRoad) {
+      availableRoad.edge = payload.edgeId
+
+      const state = this.room.state
+
+      player.payToBank(state, "Brick", 1)
```

Comment:
Ístaðin fyri at brúka "Brick" string, so skalt tú brúka `CardNames` som liggur í `Card.ts`